interp/constrextern.ml

```diff
@@ -736,8 +736,10 @@ let extern_applied_notation n impl f args =
   if List.is_empty args then
     f.CAst.v
   else
-    let args = adjust_implicit_arguments false (List.length args) 1 args impl in
+  try
```

Comment:
This `try ... catch` was missing due to the API of adjust_implicit_arguments? Using an exception here seems overkill and prone to errors maybe.